content/packages.md

```diff
@@ -0,0 +1,386 @@
+---
+title: Building Packages
+---
+
+In Meteor, there are two environments for writing code: apps and packages. The app environment is designed for rapid iteration and does a lot for you automatically. The package environment gives you much more control and enables you to ship more easily reusable and testable code.
```

Comment:
It's difficult to find another word which conveys the same impression as environment, so how about avoidance by rewording the paragraph?

> In Meteor, you may choose to structure your code anywhere along the spectrum between monolithic and fully packaged. The monolithic end is designed for rapid iteration, tolerates a relaxed "grow as you go" structure and does a lot for you automatically. The package structure gives you much more control and enables you to ship more easily reusable and testable code.